sph/include/sph/tables.hpp

```diff
@@ -12,7 +12,7 @@ constexpr int size = 20000;
 
 //! @brief create a lookup-table for sinc(x)^sincIndex
 template<typename T, std::size_t N>
-std::array<T, N> createWharmonicLookupTable(T sincIndex)
+std::array<T, N> createWharmonicTable(double sincIndex)
```

Comment:
not sure I understand the benefit of double for sincIndex here.

Reply:
With `T sincIndex` there's type-deduction of the table based on the type of `sincIndex` which is not what is needed.
Performance is not an issue here, because it's just the construction of the lookup table.